docs/scos/dev/glue-api-guides/202200.0/managing-wishlists/managing-wishlist-items.md

```diff
@@ -0,0 +1,258 @@
+                            }
+                        ]
+                    }
+                ]
+            },
+            "prices": []
+        },
+        "links": {
+            "self": "http://glue.de.spryker.local/wishlists/b70e1073-a740-5a48-bb5e-0449a9e51d53/wishlist-items/093_24495843_98bf36f052d23f10a8a081694ad4f45e"
+        }
+    }
+}
+```
+
+</details>
+
+<a name="wishlishlist-items-response-attributes"></a>
+
+| ATTRIBUTE | TYPE | DESCRIPTION |
+| --- | --- | --- |
+| sku | String | SKU of the added to the wishlist concrete product. |
+| displayData  | Array  | Array of variables a Storefront user set up in the configurator.  |
+| configuration  | Array  | Default product configuration.  |
+| configuratorKey  | String  | Configurator type.  |
+| isComplete  | Boolean  | Shows if the configurable product configuration is complete:<div><ul><li>`true`—configuration complete.</li><li>`false`—configuration incomplete.</li></ul></div>  |
+| quantity  | Integer  | Quantity of the product that was added to the wishlist.  |
+| availableQuantity  | Integer  | Product quantity available in the store. |
+
+For attribute descriptions of product prices, see [Retrieving abstract product prices](/docs/scos/dev/glue-api-guides/{{page.version}}/managing-products/abstract-products/retrieving-abstract-product-prices.html#abstract-product-prices-response-attributes).
+
+For attribute desriptions of concrete product availabillity, see [Retrieving concrete product availability](/docs/scos/dev/glue-api-guides/202108.0/managing-products/concrete-products/retrieving-concrete-product-availability.html#concrete-product-availability-response-attributes).
+
+## Delete a wishlist item
+
+To delete wishlist item, send the request:
+
+---
+`DELETE` **/wishlists/*{% raw %}{{{% endraw %}wishlist_id{% raw %}}}{% endraw %}*/wishlist-items/*{% raw %}{{{% endraw %}item_sku{% raw %}}}{% endraw %}***
+
+---
+
+| PATH PARAMETER | DESCRIPTION |
+| --- | --- |
+| ***{% raw %}{{{% endraw %}wishlist_id{% raw %}}}{% endraw %}*** | Unique identifier of the wishlist to delete an item from. [Create a wishlist](/docs/scos/dev/glue-api-guides/{{page.version}}/managing-wishlists/managing-wishlists.html#create-a-wishlist) or [retrieve all wishlists](/docs/scos/dev/glue-api-guides/{{page.version}}/managing-wishlists/managing-wishlists.html#retrieve-wishlists) to get it. |
+| ***{% raw %}{{{% endraw %}item_sku{% raw %}}}{% endraw %}*** | Unique identifier of the product to delete. |
+
+### Request
+
+| HEADER KEY | HEADER VALUE | REQUIRED | DESCRIPTION |
+| --- | --- | --- | --- |
+| Authorization | string | &check; | Alphanumeric string that authorizes the customer to send requests to protected resources. Get it by [authenticating as a customer](/docs/scos/dev/glue-api-guides/{{page.version}}/managing-customers/authenticating-as-a-customer.html).  |
+
+Request sample: `DELETE https://glue.mysprykershop.com/wishlists/09264b7f-1894-58ed-81f4-d52d683e910a/wishlist-items/064_18404924`
```

Comment:
I think we should describe additional request with a configurable product in case.
I this case we use not just `sku`  but `sku_productConfigurationInstanceHash` to identify item with the same sku but different configuration. Schema of usage the same. Recive this identifer we can also when adding wishlist item or getting items `"id": "093_24495843_98bf36f052d23f10a8a081694ad4f45e"`

Reply:
So, basically, what I should do is to add an additional `DELELE` request, but with `sku_productConfigurationInstanceHash` instead of `wishlist_item_id`, right?

Reply:
Done.